application-task-ui/src/main/resources/TaskManager/EditorContentUpdater.xml

```diff
@@ -0,0 +1,543 @@
+        <values>long|short|default|forbid</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </cache>
+      <code>
+        <contenttype>PureText</contenttype>
+        <disabled>0</disabled>
+        <editor>PureText</editor>
+        <name>code</name>
+        <number>2</number>
+        <prettyName>Code</prettyName>
+        <restricted>0</restricted>
+        <rows>20</rows>
+        <size>50</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </code>
+      <name>
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>1</number>
+        <prettyName>Name</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <parse>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>parse</name>
+        <number>4</number>
+        <prettyName>Parse content</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </parse>
+      <use>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>use</name>
+        <number>3</number>
+        <prettyName>Use this extension</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators>|, </separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>currentPage|onDemand|always</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </use>
+    </class>
+    <property>
+      <cache>long</cache>
+    </property>
+    <property>
+      <code>require(['jquery', 'diff-dom'], function ($, diffDOM) {
+  let htmlDecode = function(content) {
+    let doc = new DOMParser().parseFromString(content, "text/html");
+    return doc.documentElement.textContent;
+  };
+  let handleCKEWysiwyg = function (content, editor) {
+    let currentHTMLContent = CKEDITOR.plugins.xwikiSource.getFullData(editor);
+    let convertURL = XWiki.currentDocument.getURL('get', $.param({
+      sheet: 'CKEditor.HTMLConverter',
+      outputSyntax: 'plain',
+      language: $('html').attr('lang') || '',
+      formToken: document.documentElement.dataset.xwikiFormToken || ''
+    }));
+    $.ajax({
+      url: convertURL,
+      method: 'POST',
+      data: {
+        sourceSyntax: 'xwiki/2.1',
+        // might be true if editing inline
+        stripHTMLEnvelope: editor.elementMode === CKEDITOR.ELEMENT_MODE_INLINE,
+        fromHTML: false,
+        toHTML: true,
+        text: content
+      }
+    })
```

Comment:
You should reuse instead ``CKEDITOR.plugins.xwikiSource.convertHTML(...)``. See https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-source/plugin.js#L161C7-L161C47 .